docs/examples/word_embedding/word_embedding.md

```diff
@@ -282,13 +289,15 @@ get_knn(vocab, 5, 'beautiful')
 ### Word Analogy
 
 We can also apply pre-trained word embeddings to the word
-analogy problem. For instance, "man : woman :: son : daughter" is an analogy.
-The word analogy completion problem is defined as: for analogy 'a : b :: c : d',
-given teh first three words 'a', 'b', 'c', find 'd'. The idea is to find the
+analogy problem. For example, "man : woman :: son : daughter" is an analogy.
+This sentence can also be read as "A man is to a woman as a son is to a daughter."
+
+The word analogy completion problem is defined concretely as: for analogy 'a : b :: c : d',
+given the first three words 'a', 'b', 'c', find 'd'. The idea is to find the
 most similar word vector for vec('c') + (vec('b')-vec('a')).
 
 In this example,
-we will find words by analogy from the 400,000 indexed words in `vocab`.
+we will find words that are analogous from the 400,000 indexed words in `vocab`.
```

Comment:
Something like "...words that complete each analogy from..." to be more explicit?

Reply:
i think we're plenty specific with the example? And I was just trying to change the grammar here.